src/core/lib/security/credentials/fake/fake_credentials.c

```diff
@@ -88,6 +92,26 @@ grpc_server_credentials *grpc_fake_transport_security_server_credentials_create(
   return c;
 }
 
+grpc_arg grpc_fake_transport_expected_targets_arg(
+    const char *expected_targets) {
+  grpc_arg arg;
+  arg.type = GRPC_ARG_STRING;
+  arg.key = GRPC_ARG_FAKE_SECURITY_EXPECTED_TARGETS;
+  arg.value.string = gpr_strdup(expected_targets);
```

Comment:
It might be better to not use `gpr_strdup()` here.  I would expect the common use-case to be that the `grpc_arg` returned by this function is immediately passed to `grpc_channel_args_copy_and_add()` or `grpc_channel_args_copy_and_add_and_remove()`, in which case the string passed into this function will still be allocated before those functions are called.  Not calling `gpr_strdup()` here will avoid the need for the caller to invoke `gpr_free()` on the string from the returned `grpc_arg`.